spec/playload/repositories/base_repository_spec.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
-RSpec.describe UserRepository, type: :repository do
+RSpec.describe BaseRepository, type: :repository do
```

Comment:
Should we not still test that the user repository implements the base methods? Or are we going to rely on the assumption that all repositories include base repository methods?

Reply:
The latter. Perhaps we can write a separate test instead that just makes sure every repo has base_repo included. That would probably be a good idea actually.